controllers/homeController.js

```diff
@@ -19,15 +19,13 @@ router.post('/', function(req, res) {
             var summonerStats = results[0]
             var summonerStats2 = results[1]
             var commonChampions = findCommonChampions(summonerStats, summonerStats2)
-            console.log(commonChampions);
 
             var compareChampionsResult = compareCommonChampions(commonChampions);
             displayComparisonResults(summonerName, summonerName2, compareChampionsResult);
-            res.render('results', { winner: compareChampionsResult.compare, resultList:compareChampionsResult.championCompareList });
-
+            res.render('results', { summonerA: summonerName, summonerB: summonerName2, winner: compareChampionsResult.compare, resultList:compareChampionsResult.championCompareList });
```

Comment:
Okay. The intent of that function was to display the results in whatever fashion we would be doing so. But if it's just going to be a single call to render, then not having a separate function for it is fine.